protos/route.proto

```diff
@@ -37,6 +37,31 @@ message CreateCustomSyncResponse {
   bool ok = 4;
 }
 
+message AlertConfig {
+  int32 id = 1;
+  string service_type = 2;
+  string service_config = 3;
+}
+message GetAlertConfigsRequest {
+  int32 id = 1;
+}
+message PostAlertConfigRequest {
+  int32 id = 1;
+  string service_type = 2;
+  string service_config = 3;
+}
+message DeleteAlertConfigRequest {
+  int32 id = 1;
+}
+message GetAlertConfigsResponse {
+  repeated AlertConfig configs = 1;
+}
+message PostAlertConfigResponse {
+  int32 id = 3;
+}
+message DeleteAlertConfigResponse {
+}
```

Comment:
should contain `ok` at the very least, same for other routes

Reply:
seems like errors should rely on grpc error reporting mechanism already there